oas_docs/output/kibana.serverless.yaml

```diff
@@ -45082,8 +45082,8 @@ components:
       type: string
     Security_Endpoint_Exceptions_API_NonEmptyString:
       description: A string that is not empty and does not contain only whitespace
+      format: nonempty
```

Comment:
I'm curious how this is going to be displayed in bump.sh docs

Reply:
I deployed the result OAS bundle to Bump.sh and got this

![image](https://github.com/user-attachments/assets/b91aa502-6cd4-4388-b326-2d30da9069fb)

It adds format in parenthesis resulting in `STRING(NONEMPTY)`.